yarn2nix.nix

```diff
@@ -11,6 +11,7 @@ let
     mainnet = 8090;
     staging = 8092;
     testnet = 8094;
+    jormungandr-testnet = 8190;
```

Comment:
this table should probably be removed, and fully random ports put back into effect